server/world/entity.go

```diff
@@ -67,3 +67,8 @@ func EntityByName(name string) (SaveableEntity, bool) {
 	e, ok := entities[name]
 	return e, ok
 }
+
+// Entities returns all registered entities.
+func Entities() map[string]SaveableEntity {
```

Comment:
I'd make this grab the values from the map and return a slice. We don't want people to be able to modify this map directly.